js/client/modules/@arangodb/testutils/testrunner.js

```diff
@@ -380,9 +381,7 @@ class testRunner {
     if (checkCollections) {
       this.cleanupChecks.push(graphsTest);
     }
-    this.cleanupChecks.push();
     this.instanceManager;
```

Comment:
This line is totally useless.
```suggestion
```

Reply:
we now have assigned it `undefined` for better structural readability to have all object members enlisted in the constructor.